controller/lib/core/sensors.cpp

```diff
@@ -71,7 +78,37 @@ static constexpr Duration VOLUME_INTEGRAL_INTERVAL = milliseconds(5);
   __builtin_unreachable();
 }
 
-Sensors::Sensors() {}
+// Tuning params for PID that calculates error in our flow measurements.
+// Calculated using the
+// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method The constants in
+// the constructor are for the "classic PID" control type, which seemed to work
+// well.
+//
+// Note that we have to choose a nominal sample period for the PID.  In reality
+// we sample the PID once per breath; the exact value we use for the sample
+// period is not important so long as it's shorter than our breath period.  If
+// we were to choose a value longer than our breath period, then the PID
+// wouldn't update on some breaths, which is not desirable!
```

Comment:
So in this case it goes way beyond "jitter" - real breaths are several times longer than the sample period. Is the math in the PID change still sound under these conditions?

Reply:
Added a comment indicating how I think we should change the PID.